lua/nvim-tree.lua

```diff
@@ -415,10 +426,10 @@ local DEFAULT_OPTS = { -- BEGIN_DEFAULT_OPTS
       max = vim.diagnostic.severity.ERROR,
     },
     icons = {
-      hint = "",
-      info = "",
-      warning = "",
-      error = "",
+      hint = get_diagnostics_icons(vim.diagnostic.severity.HINT) or "",
+      info = get_diagnostics_icons(vim.diagnostic.severity.INFO) or "",
+      warning = get_diagnostics_icons(vim.diagnostic.severity.WARN) or "",
+      error = get_diagnostics_icons(vim.diagnostic.severity.ERROR) or "",
```

Comment:
When the user copies and uses this default config this fails, as `get_diagnostics_icons` is local to this file.

No matter, we will work around this another way.